draft-ounsworth-cfrg-kem-combiners.mkd

```diff
@@ -143,13 +150,34 @@ informative:
       -
         ins: G. Assche
         name: Gilles van Assche
-    date: January 2011
+    date: 2008
+  KEMCombiners:
+    target: https://doi.org/10.1007/978-3-319-76578-5_7
+    title: KEM Combiners
+    date: 2018
+    author:
+      - 
+        ins: F. Giacon
+        name: Federico Giacon
+      -
+        ins: F. Heuer
+        name: Felix Heuer
+      -
+        ins: B. Poettering
+        name: Bertram Poettering
+  ETSI-hybridKEM:
+    target: https://www.etsi.org/deliver/etsi_ts/103700_103799/103744/01.01.01_60/ts_103744v010101p.pdf
+    title: Quantum-safe Hybrid Key Exchanges
+    date: 2020-12
+    seriesinfo:
+      ETSI TS 103 744 V1.1.1
 
 --- abstract
 
 The migration to post-quantum cryptography often calls for performing multiple key encapsulations in parallel and then combining their outputs to derive a single shared secret.
 
-This document defines a flexible multi-share KEM combiner to join an arbitrary number of key shares, that is a multi-PRF compatible with NIST SP 800-56Cr2 [SP800-56C].
+This document defines a flexible multi-share KEM combiner to join an arbitrary number of key shares, that is a multi-PRF compatible with NIST SP 800-56Cr2 [SP800-56C]. The combiner defined here is (CCA) secure as long as at least one of the ingredient
```

Comment:
```suggestion
This document defines a flexible multi-share KEM combiner to join an arbitrary number of key shares, that is a multi-PRF compatible with NIST SP 800-56Cr2 [SP800-56C]. The combiner defined here is CCA secure as long as at least one of the ingredient
```
As I read those two sentences, there are two security claims. First sentence: multi-PRF. Second sentence: CCA secure. So the brackets around CCA can be omitted IMO.

Reply:
Added manually to my branch.